.prettierignore

```diff
@@ -11,3 +11,5 @@ testing*
 *.pyc
 bin/
 ro-crate-metadata.json
+modules/nf-core/rtgtools/bndeval/meta.yml
+modules/nf-core/rtgtools/svdecompose/meta.yml
```

Comment:
you shouldn't need this in my opinion, but I'm guessing it's just a question of modules and pipeline out of sync, so fine for me

Reply:
exactly, I had a sync problem and forgat to remove those lines..